fellowship/cli.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2021 Nokia
+# Licensed under the BSD 3-Clause License.
+# SPDX-License-Identifier: BSD-3-Clause
+
+import argparse
+import json
+import sys
+from .contract_generator import ContractGenerator
+from .rest_tester import RestTester
+
+
+def validation(args):

```

Comment:
Imperative format could be better validation --> validate. In here we really command validation.  The 'validation' would merely mean a process of validation  but possibly do not have any definite outcome. The same applies to 'generation' which could be 'generate' instead.